main.js

```diff
@@ -0,0 +1,49 @@
+console.log('Task 6');
+let user = 'John Doe';
+console.log(user);
+
+let student = 'Eryka';
+console.log(student);
+/*The 'user' declaration has the value of the 'student' declaration. But if it was "const" am error would occur*/
+user = student;
+console.log(user);
+
+let test = 1 + '1';
+/*I thought that the result would be "2". But "1" is a number and '1' is a string. They connect*/
+console.log(test);
+test = 1 - 1;
+/*Result will be "0"*/
+console.log(test);
+console.log('Task 7');
+test = true;
+console.log(test);
+
+console.log('Task 8');
+let result = '';
+let array = [2, 3, 5, 8];
+for (let i = 0; i < array.length; i++) {
+  result = result *= array[i];
```

Comment:
It is nonsensical chain of statements.
Notation `a *= b;` is a reduction for `a = a * b;`
So, when you write `result = result *= array[i];` it means `result = result = result * array[i];`
It will work the same with a lot of `result = ` preceding your statement, for example `result = result = result = result = result * array[i];`
But does it make sense?